packages/react-dom-bindings/src/shared/ReactFlightClientConfigDOM.js

```diff
@@ -140,3 +145,35 @@ export function preinitScriptForSSR(
     });
   }
 }
+
+export function preloadForSSR(
```

Comment:
Is this unused?

Reply:
at the moment, yes. I need to add more metadata to the manifest so I can determine if a chunk is an ES module or not. Right now I'm just assuming it is.